commit 32dee2159d0a22a2329cb695bc65acaac84012ea
Author: Nicholas Lim <nicknitewolf@lineageos.org>
Date:   Tue Oct 15 00:28:37 2019 +0800

    wt88047: Update fingerprint
    
     * Update to the last known CTS passing fingerprint
    
    Change-Id: I59043b3582899b9dcc8600148a5d86483c09af85

diff --git a/lineage.mk b/lineage.mk
index a00f3ca..0458ffe 100644
--- a/lineage.mk
+++ b/lineage.mk
@@ -31,5 +31,5 @@ PRODUCT_GMS_CLIENTID_BASE := android-xiaomi
 
 # Build fingerprint
 PRODUCT_BUILD_PROP_OVERRIDES += \
-    BUILD_FINGERPRINT="Xiaomi/wt88047/wt88047:5.1.1/LMY47V/V8.1.3.0.LHJMIDL:user/release-keys" \
-    PRIVATE_BUILD_DESC="wt88047-user 5.1.1 LMY47V V8.1.3.0.LHJMIDL release-keys"
+    BUILD_FINGERPRINT="Xiaomi/2014817/HM2014817:5.1.1/LMY47V/V9.2.5.0.LHJMIEK:user/release-keys" \
+    PRIVATE_BUILD_DESC="2014817-user 5.1.1 LMY47V V9.2.5.0.LHJMIEK release-keys"
